index.html

```diff
@@ -96,17 +92,17 @@ <h3>Lecture</h3>
           <div class="card-img">
             <img src="image/mpro-img3-removebg-preview.png" alt="" />
```

Comment:
- [ ]  As previous reviewer suggested, you should not left alt attribute empty  to increase your website accessibility. This will help the user to know what's was on the image place in case the images does not appear. consider doing to all alt attributes in your html code base.